release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -111,6 +111,13 @@ For more information, see xref:../networking/ingress-controller-dnsmgt.adoc#ingr
 
 For more information, see xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Supported devices].
 
+[id="ocp-4-12-ipv6-gratuitous-ARPs-default-SR-IOV"]
+==== IPv6 unsolicited neighbor advertisements and IPv4 gratuitous address resolution protocol now default on the SR-IOV Operator
+
+Pods created on a Single Root I/O Virtualization (SR-IOV)-enabled node (where the IP address management CNI plug-in has assigned IPs) now send IPv6 unsolicited neighbor advertisements and/or IPv4 gratuitous address resolution protocol by default onto the network. This enhancement notifies hosts of the new pod's MAC address for a particular IP to refresh ARP/Neighbor caches with the correct information.
```

Comment:
This is more correct: Pods created with the Single Root I/O Virtualization (SR-IOV) CNI plugin